addons/cloud_storage_google/utils/cleanup_cloud_storage_google.py

```diff
@@ -0,0 +1,131 @@
+import json
+import logging
+import requests
+import xmlrpc.client
+
+from concurrent.futures import ThreadPoolExecutor
+from google.oauth2 import service_account
+from google.auth.transport.requests import Request
+from itertools import islice
+from urllib.parse import quote
+
+# This is a script to manually clean up unused google blobs.
```

Comment:
I am worried about scaling when you have many used blobs.  If you have 100 000 blobs, and 1% of them are garbage, this will take a lot of time, won't it?